### ISSUE TITLE
Unable to install REE on El Capitan

### ISSUE THREAD

**Issue by pdornfel** [rvm/rvm#3551]
CFLAGS="-O2 -fno-tree-dce -fno-optimize-sibling-calls"
I have an el capitan machine that I am trying to get REE installed on (i know its EOL, its a legacy app that we are trying to get the ruby version upgraded on …) and getting errors, 

here is the gist of the install: 
https://gist.github.com/pdornfel/124e8b770102f49f8f54

anybody ever seen that issue?
ld: warning: directory not found for option '-L/opt/local/lib'

dyld: Library not loaded: @rpath/libsystem_allocator.dylib

Referenced from: /Users/pdornfeld/.rvm/src/ree-1.8.7- 2012.02/source/./miniruby

Reason: image not found

make: *** [.rbconfig.time] Trace/BPT trap: 5

++ return 1

here is a link to the full install log:
https://gist.github.com/pdornfel/333bc7681011a41b8300

**Comment by ahoym** [comment]
Bump, I'm having this exact same issue as well.

**Comment by tcocca** [comment]
@ahoym Any luck w/ this?  We still haven't gotten it to work yet.

**Comment by tcocca** [comment]
@ahoym This issue may shed some light: https://github.com/sstephenson/ruby-build/issues/815 specifically here: https://github.com/sstephenson/ruby-build/issues/815#issuecomment-153199147

RVM will install openssl098 automatically for an ree install from brew though (in my experience) and I'm not sure you need the brew link openssl098 --force / brew unlink openssl098 if you just pass the --with-openssl-dir param through when install ree but I'm not 100% positive.

Looks like it might be due to El Capital System Integrity Protection.  I have not tried disabling, installing ree through rvm and then re-enabling yet.  Might try later, but it seems that the issue is where it is trying to install its own libsystem_allocator in /opt/local/lib but the el capitan rootless is preventing that from being created. Might be worth a shot to disable it install and re-enable.

**Comment by ahoym** [comment]
@tcocca yes I believe this is because of El Capitan's System Integrity Protection feature.

Later that day, I also stumbled across that comment and tried it out after some research. It worked.

**Comment by pdornfel** [comment]
:+1: https://github.com/sstephenson/ruby-build/issues/815#issuecomment-153199147

installing ree using rvm after disabling SIP worked.Thank you.

**Comment by Hates** [comment]
I don't believe this works anymore as `openssl098` has recently been removed which seems to be needed during the ree install :cry:

**Comment by tcocca** [comment]
See here maybe ... haven't tried it yet actually:
https://github.com/rbenv/ruby-build/issues/815#issuecomment-182476963
